crates/daemon/src/wallet.rs

```diff
@@ -526,9 +606,112 @@ mod tests {
         async fn stopped(self) -> Self::Stop {}
     }
 
+    pub async fn create_random_folder() -> Result<PathBuf> {
+        // create temporary folder
+        let random_folder: String = thread_rng()
+            .sample_iter(&Alphanumeric)
+            .take(7)
+            .map(char::from)
+            .collect();
+        let data_dir = env::temp_dir().join(Path::new(&random_folder));
+        tokio::fs::create_dir_all(&data_dir).await?;
+
+        Ok(data_dir)
+    }
+
+    #[tokio::test]
+    async fn import_itchysats_seed_with_invalid_seed_length() {
+        let data_dir = create_random_folder()
+            .await
+            .expect("could not create random temp folder");
+
+        let mut tasks = Tasks::default();
+
+        let database = sled::open(data_dir.clone()).expect("could not open database");
+        let database = database
+            .open_tree("wallet name")
+            .expect("could not open tree");
+
+        // create wallet with only one UTXO which will be locked for a
+        // long time after being used
```

Comment:
🔧 This comment seems to be irrelevant for this test.

Reply:
Probably better if you set up the test with an empty wallet, since it's not relevant for what you are testing.

Reply:
did not add that intentional.